euth/phases/models.py

```diff
@@ -30,6 +31,13 @@ def __str__(self):
     def content(self):
         return content[self.type]
 
+    def clean(self):
+        if (self.end_date < self.start_date):
+            raise ValidationError({
+                'end_date': 'End date can not be smaller than the start date.'
```

Comment:
If you want to you can probably also specify boths fields here. Also this string should be translated using ugettext